pickATrip_django_apps/templates/pickATrip_django_apps/base.html

```diff
@@ -21,20 +21,39 @@
 			</div>
 		{% endfor %}
 	{% endif %}
+
 	
 	<div id ="content" class = "container">
 		{% block content %}{% endblock %}
 	</div>
-	<!-- Optional JavaScript -->
-    <!-- jQuery first, then Popper.js, then Bootstrap JS -->
-    <script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" 
-	integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" 
-	crossorigin="anonymous"></script>
-    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.12.9/umd/popper.min.js" 
-	integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" 
-	crossorigin="anonymous"></script>
-    <script src="https://maxcdn.bootstrapcdn.com/bootstrap/4.0.0/js/bootstrap.min.js" 
-	integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" 
-	crossorigin="anonymous"></script>
+
+	<nav class="navbar navbar-expand-lg navbar-light bg-light">
+  <div class="container-fluid">
+    <a class="navbar-brand" href="#">
+	</a>
+    <button class="navbar-toggler" type="button" data-bs-toggle="collapse" data-bs-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+      <span class="navbar-toggler-icon"></span>
+    </button>
+    <div class="collapse navbar-collapse" id="navbarNav">
+      <ul class="navbar-nav">
+        <li class="nav-item">
+          <a class="nav-link " aria-current="page" href="#">Home</a>
+        </li>
+		<li class="nav-item">
+          <a class="nav-link " aria-current="page" href="#">About us</a>
+        </li>
+      </ul>
+    </div>
+  </div>
+</nav>
+
+	
+		<footer class="footer mt-auto py-3 bg-light" style="position: fixed;width: 100%; bottom : 0;>
+  <div class="container">
+    <span class="text-muted" style="padding:2px; margin: 4px ">Amir Miller, Shoval Ovadia,Leead Arbetman, Ma'ayan Hadar,Nadav Spinco</span>
```

Comment:
Please add spaces after each comma (I mean the - ','), to make it consistent, as you did partialy

Reply:
done.